docs/section16/Rule 16-6.md

```diff
@@ -0,0 +1,33 @@
+# Elevators – Rule 16-6  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-6  
+**Rule Description:** When included in the proposed design, the baseline elevator cab ventilation fan power shall be 0.33 W/cfm  
+**Rule Assertion:** B-RMD = Expected Value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16  
+**Data Lookup:** None  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+N/A
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- For each elevator in the baseline RMD: `for elevator in B_RMD...elevators`
+  - Get the elevator cab ventilation fan power: `elevator_cab_ventilation_fan_power_b = elevator.cab_ventilation_fan_power`
+  - Get the elevator cab ventilation fan airflow: `elevator_cab_ventilation_fan_flow_b = elevator.cab_ventilation_fan_flow`
```

Comment:
I know we have discussed this a million times, just want to verify that it will be clear to the RCT development team that you are expected CFM since the schema is in metric.

Reply:
The comparison with the expected value will also consider the units. I added units to the rule assertion to make sure these don't get lost in translation